features.json

```diff
@@ -14,7 +14,8 @@
     "new-computed-syntax": null,
     "ember-testing-checkbox-helpers": null,
     "ember-metal-stream": null,
-    "ember-htmlbars-each-with-index": true
+    "ember-htmlbars-each-with-index": true,
+    "ember-initializer-context": null
```

Comment:
We typically like to use the package name as the prefix. Can you update to `ember-application-initializer-context`?